plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/model/DatabricksMetaModel.java

```diff
@@ -76,6 +78,38 @@ public GenericSchema createSchemaImpl(
         return new DatabricksSchema(dataSource, catalog, schemaName);
     }
 
+    @Override
+    public List<GenericSchema> loadSchemas(
+        JDBCSession session,
+        GenericDataSource dataSource,
+        GenericCatalog catalog
+    ) throws DBException {
+        List<GenericSchema> schemas = super.loadSchemas(session, dataSource, catalog);
```

Comment:
Over engineering.
If the problem is only in missing information_schema then just add it in Java, not need to use extra selects or fix anything else.

Reply:
There's a problem that information_schema may be absent in some catalogs. If we add it manually in java there will be an exception